Fix lint and dependency issues in D67108845

YIWENX14 opened the pull request:
Differential Revision: D67108845

kirklandsign commented:
```
>>> Lint for devtools/bundled_program/test/test_bundle_data.py:

  Warning (UFMT) format
    Run `lintrunner -a` to apply this patch.

    You can run `lintrunner -a` to apply this patch.

    74  74 |             bundled_program.serialize_to_schema().program,
    75  75 |             bytes(_serialize_pte_binary(executorch_program.executorch_program)),
    76  76 |         )
    76     |-        
        77 |+
    78  78 |     def test_bundled_program_from_pte(self) -> None:
    79  79 |         executorch_program, method_test_suites = get_common_executorch_program()
    80  80 | 
--------------------------------------------------------------------------------
    84  84 |                 f.write(executorch_program.buffer)
    85  85 | 
    86  86 |             bundled_program = BundledProgram(
    86     |-                executorch_program=None,
    87     |-                method_test_suites=method_test_suites,
        87 |+                executorch_program=None,
        88 |+                method_test_suites=method_test_suites,
    89  89 |                 pte_file_path=executorch_model_path,
    90  90 |             )
    91  91 | 

```

YIWENX14 committed:
Fix lint and dependency issues in D67108845 (#7294)

Summary:
pytorchbot label "topic: not user facing"




1. fix linter issue
2. remove `pyre_extensions` dependency

Differential Revision: D67108845

kirklandsign commented:
@pytorchbot label "topic: not user facing"

YIWENX14 commented:
The issues are already fixed by PR #7296. Deleting the branch.
